include/TMU.hpp

```diff
@@ -185,7 +229,41 @@ class TMU {
             .Data = CO_LINK(0x2100, 3, 16),
         },
 
-        /**
+        /*
+         * TPDO 1 Mapping
+         * 0. The number of mapping objects in the second TPDO
+         * 1. Link to the first error - err_arr[0]
+         * 2. Link to the second error - err_arr[1]
+         * 3. Link to the third error - err_arr[2]
+         * 4. Link to the fourth error - err_arr[3]
+         */
+        {
+            .Key = CO_KEY(0x1A00, 5, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 4,
+        },
+        {
+            .Key = CO_KEY(0x1A00, 6, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = CO_LINK(0x2101, 0, 8),
```

Comment:
Since you fixed the section below, you need to fix this as well
```suggestion
            .Data = CO_LINK(0x2100, 0, 8),
```

Reply:
There are more issues with this in the lines following this. Also, please validate that the data is being reported properly over CAN before you merge this